integration-test/src/test/java/org/jboss/pnc/integration_new/endpoint/BuildEndpointTest.java

```diff
@@ -96,42 +165,119 @@ public void shouldSetDependentArtifacts() throws RemoteResourceException {
         Assertions.assertThat(updatedArtifactIds).contains(102);
     }
 
+    @Test
+    public void shouldGetSCMArchiveLink() throws ClientException, ReflectiveOperationException {
+        BuildClient client = new BuildClient(RestClientConfiguration.asAnonymous());
+
+        // Disable redirects so we can test the actual response
+        Field f =  ClientBase.class.getDeclaredField("client"); //NoSuchFieldException
```

Comment:
Do the Exceptions in comments have some meaning in the context of this test?

Reply:
Grammar police